### PR TITLE
238/vertical/import-update-puppet-scripts

### PR DESCRIPTION
**Pull request by mewitte**
#238

### CODE REVIEW

File: app/api/v_sphere/virtual_machine.rb [hunk 1]
```diff
@@ -261,6 +270,66 @@ def status
...
+          all_users = PuppetParserHelper.read_node_file(name)
+          sudo_users = all_users['admins']
+          new_users = User.find_all_by_id(ids)
+          puppetscript = Puppetscript.node_script(name, sudo_users, new_users)
+          git_writer.write_file(name, puppetscript)
+          message = commit_message(git_writer)
+          git_writer.save(message)
+        end
+      rescue Git::GitExecuteError
+        { alert: 'Could not push to git. Please check that your ssh key and environment variables are set.' }
```
Comment (Adrian-St) [inline app/api/v_sphere/virtual_machine.rb]: It's a bit pointless to return the error if you're not actually displaying it.
If you don't want to display it, maybe use Rails.logger.info to write the error in the logs.

File: app/api/v_sphere/virtual_machine.rb [hunk 2]
```diff
@@ -261,6 +270,66 @@ def status
...
+          all_users = PuppetParserHelper.read_node_file(name)
+          users = all_users['users']
+          new_sudo_users = User.find_all_by_id(ids)
+          puppetscript = Puppetscript.node_script(name, new_sudo_users, users)
+          git_writer.write_file(name, puppetscript)
+          message = commit_message(git_writer)
+          git_writer.save(message)
+        end
+      rescue Git::GitExecuteError
+          { alert: 'Could not push to git. Please check that your ssh key and environment variables are set.' }
```
Comment (Adrian-St) [inline app/api/v_sphere/virtual_machine.rb]: Same here

File: app/models/user.rb [hunk 3]
```diff
@@ -118,6 +128,8 @@ def update_repository
                   end
         git_writer.save(message)
       end
+    rescue Git::GitExecuteError
+      { alert: 'Could not push to git. Please check that your ssh key and environment variables are set.' }
```
Comment (Adrian-St) [inline app/models/user.rb]: Same here

File: app/api/v_sphere/virtual_machine.rb (hunk 1 above)
Comment (NiklasKoehnecke) [reply app/api/v_sphere/virtual_machine.rb]: this is handeled by #357

**Pull request merged**